lib/bdev/rbd/blockdev_rbd.c

```diff
@@ -429,6 +431,18 @@ blockdev_rbd_free_channel(struct blockdev_rbd_io_channel *ch)
 	}
 }
 
+static int blockdev_rbd_handle(void *arg)
+{
+	struct blockdev_rbd_io_channel *ch = (struct blockdev_rbd_io_channel *)arg;
+
```

Comment:
Our coding style is to group all variable declarations at the top of the scope with no blank lines in between, and then leave one blank line between variable declarations and code.